backend/src/models/game.model.ts

```diff
@@ -16,14 +16,14 @@ export enum GameState {
 export interface Game extends Document {
   gameCode: string;
   settings?: Settings;
-  setups: Setup[];
-  discardedSetups?: Setup[];
+  setups: Types.DocumentArray<Setup>;
```

Comment:
Yeah TypeScript needed subdocuments to be typed using `mongoose.Types.DocumentArray<Type>` instead of `Type[]` to properly use them in services (as arrays subdocuments have additional methods such as `id()`)